service/src/test/java/bio/terra/pipelines/testutils/TestUtils.java

```diff
@@ -224,4 +223,8 @@ public static PipelineRun createNewPipelineRunWithJobIdAndUser(UUID jobId, Strin
         CONTROL_WORKSPACE_GOOGLE_PROJECT,
         CommonPipelineRunStatusEnum.PREPARING);
   }
+
+  public static String buildResultUrl(String jobId) {
```

Comment:
super nit, but for ease of reading tests and knowing this is a test method and not one used in the service, could we rename this to `buildTestResultUrl`?